drafthorse/models/party.py

```diff
@@ -129,7 +129,7 @@ class TradeParty(Element):
         profile=BASIC,
         _d="Anschrift des Handelspartners",
     )
-    electronic_address: Container = MultiField(
+    electronic_address: Container = Field(
```

Comment:
```diff
- electronic_address: Container = Field(
+ electronic_address: URIUniversalCommunication = Field(
```

@Ayuto the type hint also needs to be adjusted, since `Field` is not a `Container`.